ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -160,39 +164,42 @@ const svGenotype = (genotype, isHemiX) => {
   )
 }
 
+const AllelesHeader = ({ genotype, variant, isHemiX }) => (
+  <Header.Content>
+    <Allele isAlt={genotype.numAlt > (isHemiX ? 0 : 1)} variant={variant} textAlign="right" />
+    /
+    {isHemiX ? '-' : <Allele isAlt={genotype.numAlt > 0} variant={variant} textAlign="left" />}
+    {genotype.mitoCn && (copyNumberGenotype(genotype.mitoCn, true))}
+  </Header.Content>
+)
+
+AllelesHeader.propTypes = {
+  genotype: PropTypes.object,
+  variant: PropTypes.object,
+  isHemiX: PropTypes.bool,
+}
+
 export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) => (
   <AlleleContainer>
     {warning && (
       <Popup
         wide
         trigger={<WarningIcon />}
-        content={
```

Comment:
so as mentioned on a previous PR, this is an exported component so its better to its behavior alone. I also think it actually better for us to say "warning" one time in the popup, not once per warningas repeating the word multiple time reduces readability